packages/dataviews/src/dataviews-layouts/list/index.tsx

```diff
@@ -359,10 +361,11 @@ export default function ViewList< Item >( props: ViewListProps< Item > ) {
 
 	const store = useCompositeStore( {
 		defaultActiveId: getItemDomId( selectedItem ),
-	} );
+	} ) as CompositeStore; // TODO, remove once composite APIs are public
```

Comment:
Necessary for the `useStoreState` call. Previously was `any` due to the locked nature of the private API.